src/ResourceGraph/ResourceGraph/help/Search-AzGraph.md

```diff
@@ -74,7 +75,7 @@ id         : /subscriptions/1ef51df4-f8a9-4b69-9919-1ef51df4eff6/resourceGroups/
 name       : egtopic-2
```

Comment:
please validate that the cmdlet results are in this format as well with the new changes.